docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -29,6 +29,230 @@ can be run multiple times) and that it can be run on multiple units simultaneous
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
 
+Troubleshoot the charm
+----------------------
+
+This guide provides helpful commands to run in situations where your web app
+deployment is not successful (for instance, if your app crashes).
+
+Get the app logs
+~~~~~~~~~~~~~~~~
+
+The first step of troubleshooting is to review the logs for errors. They are
+your first course of action because they flag simple errors caught in the
+stack, but they are also a constant touchpoint when diagnosing and debugging
+issues. You'll first need to obtain the logs for your particular framework
+using Pebble.
+
+To view the Pebble logs for a deployed web app, run:
+
+.. tabs::
+
+    .. group-tab:: Django
+
+        .. code-block:: bash
+
+            juju ssh --container django-app <django-app-name>/0 pebble logs
+
+    .. group-tab:: FastAPI
+
+        .. code-block:: bash
+
+            juju ssh --container app <fastapi-app-name>/0 pebble logs
+
+    .. group-tab:: Flask
+
+        .. code-block:: bash
+
+            juju ssh --container flask-app <flask-app-name>/0 pebble logs
+
+    .. group-tab:: Go
+
+        .. code-block:: bash
+
+            juju ssh --container app <go-app-name>/0 pebble logs
+
+If your app is :ref:`integrated with the Canonical Observability
+Stack <integrate_web_app_cos>`, then these logs will be sent to Grafana or Loki
+and can be viewable in the web app dashboard.
```

Comment:
Only for Flask and Django are they available on the dashboard, for the others you have to click on explore and pick loki I think